app/src/shared/errors/Web3/index.jsx

```diff
@@ -2,10 +2,24 @@
 
 import { I18n } from 'react-redux-i18n'
 
+export const errorCodes = {
```

Comment:
I'm wondering about naming conventions here. Structures like this behave like enums. I'd like to use singular for them (like I did to `NotificationIcon`), and make the name of it look "nice", e.g. `ErrorCode.WEB3_NOT_SUPPORTED`. Any thoughts?

Reply:
I was wondering the same and if it's better to keep these in constants or in the associated files. But probably enum type naming is a good idea.